HW2.Rmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "HW2"
+author: "Roxanne Felig"
+date: "2/7/2021"
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Assignment 02: Explore gapminder and use dplyr
+
+```{r load packages}
+# load your packages here:
+library(gapminder)
+library(tidyverse)
+library(dplyr)
+```
+
+### 1.1 Use filter() to subset the gapminder data to three countries of your choice in the 1970’s.
+```{r filter}
+gapminder %>%
+  filter(country %in% c("Argentina","Chile","Germany") & year>1969 & year<1980)
+```
+
+###1.2 Use the pipe operator %>% to select “country” and “gdpPercap” from your filtered dataset in 1.1.
+
+```{r select}
+gapminder %>%
+  filter(country %in% c("Argentina","Chile","Germany") & year>1969 & year<1980) %>% 
+  select(country,gdpPercap)
+
+```
+###1.3 Make a new variable in gapminder for the change in life expectancy from the previous measurement. Filter this table to show all of the entries that have experienced a drop in life expectancy.
+Hint: you might find the lag() or diff() functions useful.
+
+```{r new variable}
+gapminder %>%
+  group_by(country) %>% 
+  mutate(lifeExp_change=lifeExp-lag(lifeExp)) %>% 
+  filter(lifeExp_change<0)
+
+```
+
+###1.4 Filter gapminder so that it shows the max GDP per capita experienced by each country.
+Hint: you might find the max() function useful helpful
+
+```{r maxGDP}
+gapminder %>%
+  group_by(country) %>% 
+  summarize(maxgdp=max(gdpPercap))
```

Comment:
This is also enlightening, I used this "filter(gdpPercap == max(gdpPercap)". I wonder if they do the same thing.